app/cdap/text/text-en.yaml

```diff
@@ -3439,3 +3439,13 @@ features:
   ConnectionsList:
     labels:
       loadToGrid: 'Wrangle'
+  FooterPanel:
+    labels:
+      and: and
+      columns: columns
+      columnViewPanel: Column View Panel
```

Comment:
we need to discuss this, not sure if this is appropriate way to handle and

Reply:
Done